### PR TITLE
Release 3.0 beta 1

### PR DESCRIPTION
**Pull request by CharliePoole**
Ready for review. This will need to be merged once we do the release.

### CODE REVIEW

File: docs/3.0/vsTestAdapter.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<h2>NUnit 3.0 Test Adapter for Visual Studio</h2>
+
+<p>The NUnit 3.0 Test Adapter allows you to run NUnit 3.0 tests inside Visual Studio. This is a new adapter, based partly on the code of the original NUnit Test Adapter, but modified to work with NUnit 3.0.</p>
+
+<p>It is not possible to run NUnit 2.x tests using this adapter. Use the original adapter for that purpose. If you need to work with projects using NUnit 2.x and other projects using NUnit 3.0, you may install both versions of the adapter.</p>
+
+<p>The current 3.0 Beta release is designed to work with Studio 2012 (All updates), and with Visual Studio 2013 (All updates including Update 3 checked September 2014). Some features are not available under VS2012 RTM.</p>
```
Comment (rprouse) [inline docs/3.0/vsTestAdapter.html]: Should this read (add Visual in middle and delete last sentence), "The current 3.0 Beta release is designed to work with Visual Studio 2012 (All updates), and with Visual Studio 2013 (All updates including Update 3 checked September 2014)."

Also, I believe it now works on VS 2015 and has been checked on the latest VS 2013 update which is 4?

File: docs/3.0/vsTestAdapter.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+<li>Use the Extension Manager</li>
+<li>Add it as a Nuget package to your solution. (Requires VS 2012 Update 1 or later)</li>
+</ol>
+
+<h3>Installing With the Extension Manager</h3>
+
+<p>To install the NUnit Test Adapter using the Extension Manager, follow these steps:</p>
+
+<ol>
+<li>From within Visual Studio 2012 or 2013, select Tools | Extension Manager.</li>
```
Comment (rprouse) [inline docs/3.0/vsTestAdapter.html]: From within Visual Studio 2012 or later? Future proof :)

File: docs/3.0/vsTestAdapter.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+<li>Click 'Install'</li>
+<li>In the "Select Projects" dialog, you need to select at least one project to add the adapter to, see notes below.</li>
+</ol>
+
+<h3>Notes</h3>
+
+<p><strong>Note 1: How to choose between Extension and NuGet package</strong></p>
+
+<p>The Extension will apply to Visual Studio itself, and will work for all projects you use. All users of your solution need to install the Extension. If you use TFS Build you must also install the extension to the build system there.
+The Package will apply to the solution, and will work for any other user too, as it follows the solution, but requires the user to have Update 1 or above. It will also work for any TFS2012 Update 1 or above server build, including TF Service and requires no further installation.</p>
```
Comment (rprouse) [inline docs/3.0/vsTestAdapter.html]: This should read "The NuGet Package"?

Should it be after the next heading?

File: docs/3.0/vsTestAdapter.html (hunk 1 above)
Comment (CharliePoole) [inline docs/3.0/vsTestAdapter.html]: I agree except for dropping the last sentence. That's still true although few folks should be running it.

File: docs/3.0/vsTestAdapter.html (hunk 2 above)
Comment (CharliePoole) [inline docs/3.0/vsTestAdapter.html]: Or just Visual Studio.

File: docs/3.0/vsTestAdapter.html (hunk 3 above)
Comment (CharliePoole) [inline docs/3.0/vsTestAdapter.html]: I'll fix the first. I think it belongs here as counterpoint to the previous paragraph.

**Comment by CharliePoole** [comment]
The binaries created from this branch are available at http://nunit.org/downloads/preview

**Comment by rprouse** [comment]
I have tested the installers and the binaries, but not the NuGet packages.
I entered an issue about Windows SmartScreen blocking the install, but I
don't think we should hold up the release to fix it. I think we should
consider that for the final release if then.

I created a test project and a number of tests and then used the console to
run them. Everything went fine.

On 25 March 2015 at 15:55, CharliePoole <notifications@github.com> wrote:

> The binaries created from this branch are available at
> http://nunit.org/downloads/preview
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit.org/pull/30#issuecomment-86193028>.
>



-- 

Rob Prouse



I welcome VSRE emails. Learn more at http://vsre.info/

**Comment by rprouse** [comment]
I created a test source for the nuget packages. All look and work correctly. I installed all three NuGet packages and ran my test project with both the MSI installed version of the console and the one installed with NuGet.

image

**Comment by CharliePoole** [comment]
Thanks for testing Rob... I will fix the release dates and then merge...

**Pull request merged**